dti-website/src/pages/team.tsx

```diff
@@ -24,13 +24,13 @@ export default function TeamPage(): JSX.Element {
     role = '',
     isLead = false
   ): readonly { info: NovaMember; id: string }[] =>
-    members
+    (members as NovaMember[])
```

Comment:
why this cast?

Reply:
i think this and the other casts are unnecessary actually

Reply:
oh just kidding, without these casts i was getting some type errors but not sure if there is a better way to resolve that

Reply:
I think it's caused by the `hello DTI` in the member. Deleting that and things should work without casts.